durablefunctionsmonitor.react/src/states/az-func-as-a-graph/buildFunctionDiagramCode.ts

```diff
@@ -9,19 +9,40 @@ function getTriggerBindingText(binding: any): string {
 
     switch (binding.type) {
         case 'httpTrigger':
+
             return `${binding.authLevel === 'anonymous' ? '#127760;' : '#128274;'} http${!binding.methods ? '' : ':[' + binding.methods.join(',') + ']'}${!binding.route ? '' : ':' + binding.route}`;
+
         case 'blobTrigger':
-            return `${space}blob:${binding.path ?? ''}`;
+
+            const blobPath = binding.blobPath ?? (binding.path ?? '');
+            
+            return `${space}blob:${blobPath}`;
```

Comment:
What are these returned values (e.g. `${space}blob:${blobPath}`) used for?

Reply:
These values are used as node captions on the diagram.
E.g. 
![image](https://user-images.githubusercontent.com/5447190/186631059-dd1aaa14-74a2-499e-81a5-bfc69687b61a.png)